### ISSUE TITLE
Doesn't open some tables anymore

### ISSUE THREAD

**Issue by JCKodel** [microsoft/AzureStorageExplorer#5400]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.22.1

### Regression From

Yes, don't know which version was previous

### Architecture

x64

### Storage Explorer Build Number

20220122.3 

### Platform

Windows

### OS Version

Windows 10

### Bug Description

I use Azure Tables.
One specific table is not opening (even letting Storage Explorer working for hours).
It just opens the table, with a loading spinning wheel.
From this storage account, only this table is not working (the other 2 are normal).
From another storage account, no problems whatsoever (even with larger tables).

### Steps to Reproduce

Hard to reproduce, since it seems only one table is affected (I can share a read-only key, if it helps)

### Actual Experience

Table never loads.

### Expected Experience

To actually work. ¬¬

### Additional Context

<details>
<summary>Table Log</summary>
```
[2022-02-03T11:04:24.047Z] (se-table-extension_19396) <INFO> API call for TableServiceClient.listTables succeeded
GET https://REDACTED.table.core.windows.net/Tables - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 0b2bc516-5ab2-4025-9b2f-b5af5a693147, Date sent: Thu, 03 Feb 2022 11:04:23 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:04:52.564Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers() - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 937233ec-e913-407c-a384-fe6828f91c62, Date sent: Thu, 03 Feb 2022 11:04:25 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:05:21.668Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MA--&NextRowKey=1!40!MEJDU0g2UHkyQ1hZTHlyd2tFQWtON094dkN3Mg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: f64d62f5-4b22-49e4-b101-703d29876e48, Date sent: Thu, 03 Feb 2022 11:04:52 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:05:52.399Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MA--&NextRowKey=1!40!ME5MZ0RzM2s5U2FaUjFXQjFyVlVmZzRNcWRQMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 329c77cf-8df2-4a61-a847-37782ec33bb9, Date sent: Thu, 03 Feb 2022 11:05:21 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:16.536Z] (se-table-extension_19396) <INFO> API call for TableServiceClient.listTables succeeded
GET https://REDACTED.table.core.windows.net/Tables - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: c2bbdf10-afc2-4e04-982a-5e417a03c1cf, Date sent: Thu, 03 Feb 2022 11:06:15 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:29.429Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Users() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: d595e4cf-5601-4d4f-a146-7a3811ba5241, Date sent: Thu, 03 Feb 2022 11:06:20 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:31.999Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Reviews() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 59032a7b-d8a2-41a5-bb6f-d8d603d94923, Date sent: Thu, 03 Feb 2022 11:06:25 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:35.397Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MA--&NextRowKey=1!40!MFlCemtiQkY3NWFMZUlrcnZXbFgwdTlPRlZtMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: f348da26-6934-4140-b559-5fd3f65d485d, Date sent: Thu, 03 Feb 2022 11:05:52 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:38.433Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Reviews() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 2ca8a6d8-0c70-4979-a60d-8fea979514dd, Date sent: Thu, 03 Feb 2022 11:06:32 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:06:40.877Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Users() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: ebdf07a1-4527-4653-ab31-d36790e4f797, Date sent: Thu, 03 Feb 2022 11:06:29 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:07:04.917Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MA--&NextRowKey=1!40!MGlycUFlVldlTVRMUFFFMVR6amU2WHdIQTRzMQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: f2563cc2-e8d8-4c06-b493-945a39936f77, Date sent: Thu, 03 Feb 2022 11:06:35 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:07:22.557Z] (se-table-extension_19396) <INFO> API call for TableClient.addEntity succeeded
POST https://REDACTED.table.core.windows.net/Reviews - 204
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 29a8ba7e-662d-4ffe-a1b5-581d0e3c294f, Date sent: Thu, 03 Feb 2022 11:07:21 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:07:33.262Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MA--&NextRowKey=1!40!MHU0WVJEM1lNWU41Z2RjdHV4S0tlU1Rma2hzMQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 8fe80691-756f-4a6d-9547-2216f34d6bd6, Date sent: Thu, 03 Feb 2022 11:07:05 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:08:01.389Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MQ--&NextRowKey=1!40!MTVlS2hGVUg0Z1l1dVJrY1U4enBNSEZOdm9nMQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 8b028c87-8b68-4007-9b01-bcdc02a19739, Date sent: Thu, 03 Feb 2022 11:07:33 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:08:18.599Z] (se-table-extension_19396) <INFO> API call for TableClient.addEntity succeeded
POST https://REDACTED.table.core.windows.net/Reviews - 204
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 26d0e402-3b2b-453e-b7af-13cb4e9ca44f, Date sent: Thu, 03 Feb 2022 11:08:18 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:08:33.132Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MQ--&NextRowKey=1!40!MUd5UmZiM1I5cE9rTXNsaEhuRXpuYU5sbzFEMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: a6aae4d9-9309-4d3a-9c3d-ae282dfa25ed, Date sent: Thu, 03 Feb 2022 11:08:01 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:09:03.074Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MQ--&NextRowKey=1!40!MVQ3cXF2WnNkamV3TFFpTmdxUmRVbXhsUnJ1MQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 75168461-99bb-47d9-b2ed-e3fcc0ca8c27, Date sent: Thu, 03 Feb 2022 11:08:33 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:09:29.611Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MQ--&NextRowKey=1!40!MWRtdmZ0VnZQdmZDRzRPTlJ6Ujd6ZE4yWUJFMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: c9dbfd91-40c1-4085-ae06-82d851e7ec47, Date sent: Thu, 03 Feb 2022 11:09:03 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:09:56.161Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!MQ--&NextRowKey=1!40!MXBoZVJDcGhNRlNmNXhLNlY2WHY1eXhzUjBIMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 840f843c-c084-4bd5-ac67-ad878e2b4977, Date sent: Thu, 03 Feb 2022 11:09:29 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:10:25.073Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!MjFCdjFHak4zZGJ0UXlnZ2ZlZEZWcTNybXByMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 4e0fd620-6ebb-41a7-9d30-f57b0b2f3714, Date sent: Thu, 03 Feb 2022 11:09:56 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:10:52.010Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!MkJzWmRqYlJNVmJHU2pnUlhBWWRlSFliT3diMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 80ddfa12-4f19-4442-a362-2cb18c417949, Date sent: Thu, 03 Feb 2022 11:10:25 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:11:19.862Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!Mk5BcHJSelZ3VGRubHFxNERxNDZIY1NaekNjMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 38da8390-8629-4121-9eee-7592d595851a, Date sent: Thu, 03 Feb 2022 11:10:52 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:11:48.877Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!MmE2R1VSTzZMTk1NR1FqYURrbjNXa3dwYzVLMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 117971d7-6f7b-4009-8c08-1533cfaa94ad, Date sent: Thu, 03 Feb 2022 11:11:20 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:12:17.225Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!MmxCY3B0U2x1MmZnMGxnaUQ4RFdMYU1La0w4Mw-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: e3e88e99-09d9-4a8c-b19f-a565228180d6, Date sent: Thu, 03 Feb 2022 11:11:49 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:12:45.535Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mg--&NextRowKey=1!40!MndWbFR4cml0ZGh5RHVnZjlQR2o4emhXdk95MQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: c4aa5aec-9f77-4e0b-96e0-3e0722263288, Date sent: Thu, 03 Feb 2022 11:12:17 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:13:13.770Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mw--&NextRowKey=1!40!MzduMGN1MEszamRXM2FhaGpVcjQwTFRXbDBCMw-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 92ff60bb-f302-42fd-8a34-f8f5fc6a9835, Date sent: Thu, 03 Feb 2022 11:12:45 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:13:44.198Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mw--&NextRowKey=1!40!M0tvbkFnd1JwVlpSb09wc3V3a1czalkwUGw4Mw-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: d5c83059-0a1a-4fce-92b2-f6d27b723cb2, Date sent: Thu, 03 Feb 2022 11:13:13 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:11.096Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mw--&NextRowKey=1!40!M1Z0NVlCMURySE15eWpxRFFtY2VUMm1uNE5rMQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: a5b78a59-2952-4847-b138-ae1e9d8831a4, Date sent: Thu, 03 Feb 2022 11:13:44 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:23.845Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/GrupalErrorLog() - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 54fdc471-cade-432a-8d9f-418a5fc1c188, Date sent: Thu, 03 Feb 2022 11:14:23 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:24.244Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCErrorLog() - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 06280a63-5b0f-4a42-911e-41a1018496b1, Date sent: Thu, 03 Feb 2022 11:14:16 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:30.322Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/AdInterval() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 104ed29c-b3bd-4d8b-89fc-1dfca456b561, Date sent: Thu, 03 Feb 2022 11:14:29 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:30.902Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/AdInterval() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: d52c8503-dead-41c2-80b1-b086d5d6354e, Date sent: Thu, 03 Feb 2022 11:14:30 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:35.212Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCErrorLog() - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 81879f03-7323-4174-94cb-a4a4ca55b1a9, Date sent: Thu, 03 Feb 2022 11:14:24 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:38.601Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mw--&NextRowKey=1!40!M2hPSzhibDUwQVV3QXNsbGZWNWhnSkFqTU5XMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 2517e923-f89b-4556-a8ee-8d0861e54a0d, Date sent: Thu, 03 Feb 2022 11:14:11 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:43.435Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Users() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: 2e30de4b-4a68-47d1-a3c6-bd39a5cbf15c, Date sent: Thu, 03 Feb 2022 11:14:33 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:14:51.655Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/Users() - 200
Credential used: Account name and key - account name: REDACTED, account key: ******, keyName: undefined
RequestId: ae875084-f1e8-4966-97c1-1160b8b70ea6, Date sent: Thu, 03 Feb 2022 11:14:43 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:15:05.235Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!Mw--&NextRowKey=1!40!M3Q1U3k3TU1PWFlWMlNmdnVHMzM1MzVLVmZCMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 06f5fdec-5603-4532-97d4-32eaa7021f90, Date sent: Thu, 03 Feb 2022 11:14:38 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:15:30.533Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NA--&NextRowKey=1!40!NDRmUDQ2a1hNUU5vME1iVVdic09YdDNNeGhSMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 35e3e987-428e-440a-be84-307e93f89a79, Date sent: Thu, 03 Feb 2022 11:15:05 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:15:58.913Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NA--&NextRowKey=1!40!NEVza2V3aDVpZ1E0R0EwTWpUd3llbTBCYTg2Mw-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 8fa00e2e-7e45-4afa-8aba-76b99f81c09d, Date sent: Thu, 03 Feb 2022 11:15:30 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:16:26.723Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NA--&NextRowKey=1!40!NFF0TzF1cnUybU5UdHFQdnJTNFJBNUQ3Q291MQ-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 118d2625-421a-4ff9-8e79-9a9891ef0fbd, Date sent: Thu, 03 Feb 2022 11:15:59 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:16:54.224Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NA--&NextRowKey=1!40!NGRMSGR3RW8xMWhRVVpKUEZ2bGprU1hyUkhqMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 08c8e298-b72d-4463-ab73-0e81a437d3b1, Date sent: Thu, 03 Feb 2022 11:16:26 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:17:22.313Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NA--&NextRowKey=1!40!NHBXVVNuVFRTWGFlRGJvTE5vbWQ4em1YcExPMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 3dcc1ea8-d53e-49d1-aa69-26ecaae9922d, Date sent: Thu, 03 Feb 2022 11:16:54 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:17:49.902Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NQ--&NextRowKey=1!40!NTFUVExYSFpsc2V0TVVjaG05YzV1d1M4UWlFMg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 3d8ddf0e-8f44-4404-bc1f-2564e37014b4, Date sent: Thu, 03 Feb 2022 11:17:22 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:18:17.954Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NQ--&NextRowKey=1!40!NUNJUkl6UHZmV1A4dFBmcExjZHRkazVST3c2Mw-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 46633f02-f9da-4702-b413-25a14bc3660a, Date sent: Thu, 03 Feb 2022 11:17:50 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
[2022-02-03T11:18:47.929Z] (se-table-extension_19396) <INFO> API call for TableClient.listEntities succeeded
GET https://REDACTED.table.core.windows.net/MDCUsers()?NextPartitionKey=1!4!NQ--&NextRowKey=1!40!NU56dFh3cVlLeFNpZkFoSlJtVVd0TmNPNWM2Mg-- - 200
Credential used: Account name and key - account name: codeartbr, account key: ******, keyName: undefined
RequestId: 3c3d8f3e-0ee9-4bfe-9568-71fff230e797, Date sent: Thu, 03 Feb 2022 11:18:18 GMT, ApiVersion: 2019-02-02, ProxySettings: undefined
```
</details>

**Comment by MRayermannMSFT** [comment]
> I can share a read-only key

That would be great. Can you send it to sehelp @ microsoft .com ?

Also, what type of storage account is this table in? Classic, gpv2, etc.?

Thanks.

**Comment by craxal** [comment]
@JCKodel Looks like we have found the issue. The service doesn't always return the same number of entities, which threw off some of our paging logic. This would make Storage Explorer think it needed to keep retrieving more entities indefinitely, hence the table not appearing to load. We will make sure a fix is available for 1.23.0.

**Comment by craxal** [comment]
Shared internal build with @JCKodel, and they can confirm the fix resolves the bug.
